src/pages/Home.tsx

```diff
@@ -1,11 +1,12 @@
-import React, { Component } from 'react';
-import { Box, Container, Grid, Button, Typography, Paper, createMuiTheme, ThemeProvider } from '@material-ui/core';
+import { Component } from 'react';
+import { Grid, Button, Typography, Paper, ThemeProvider } from '@material-ui/core';
+import { createTheme } from '@material-ui/core/styles'
 import whiteBackground from "../assets/MainPageImage.png"
-import {
-  Link
-} from "react-router-dom";
-import HomeCard from '../components/DataStructureAndAlgorithms';
-import { normalize } from 'path';
+// import {
```

Comment:
Is it safe to remove this? What are the intentions of keeping it around?

Reply:
They aren't being used at the moment. We can re-add it when we need to.

Reply:
Nah, did not update package.json, I just run 'npm install' and 'npm audit fix', which only updates the package-lock.json.